src/components/FileBrowse/FileBrowse.tsx

```diff
@@ -59,15 +60,15 @@ export const FileBrowser: React.FC<FileBrowserProps> = ({
 				} p-4 my-2 text-center`}
 			>
 				<input
+					id={inputId}
 					type="file"
 					accept={allowedFileTypes.join(", ")}
 					onChange={handleFileInput}
 					className="hidden"
-					id={`file-${randomId}`}
 				/>
-				<label htmlFor={`file-${randomId}`} className="cursor-pointer">
-					<span>Click to browse or drag and drop file here (max 10MB)</span>
-				</label>
```

Comment:
An input element shouldn't have two labels, and there are labels for this file input in `src/pages/Application/Application.page.tsx` that are more appropriate.

For example, 'Resume' is more appropriate for this label:
https://github.com/KonferCA/HawkHacks-Dashboard/blob/6bd52ffc5926edb3442d2d27428676438631112a/src/pages/Application/Application.page.tsx#L417-L430